releasenotes/MinGW/ReadMe.txt

```diff
@@ -43,17 +43,12 @@ To compile and link your own code using installed library and include files
 Method 1: Explicitly set include and linking options
 $ cd <bundle>
 $ g++ -std=c++98 samples/exifprint.cpp -I/usr/local/include -L/usr/local/lib -lexiv2 -o exifprint
-$ export LD_LIBRARY_PATH="/usr/local/lib:$LD_LIBRARY_PATH"
+$ export PATH="/usr/local/bin:$PATH"
 $ ./exifprint --version
```

Comment:
[suggestion] Those instructions are about compiling and linking, so we could drop the lines which are calling the executable `exifprint`.

Reply:
Luis.  Those commands are to compile exifprint locally.  It's "proof that you can compile, link and run an application.  However, the user isn't running bin/exivprint from the bundle.

Reply:
Oh I see. Now that I have looked in more detail I see your point. Fine then :wink: